src/test/java/application/CreateUserTest.java

```diff
@@ -0,0 +1,49 @@
+package application;
+
+import cli.CliDriver;
+import adapters.CliPresenter;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.Assert.*;
+
+public class CreateUserTest {
+    CreateUser c;
+
+    /**
+     * Setting up a test User
+     */
+    @Before
+    public void setUp() {
+        c = new CreateUser();
+    }
+
+    /**
+     * Tests the getRequestModel method
+     */
+    @Test
+    public void testGetRequestModel() {
+
+        assertNotNull(c.getRequestModel());
+    }
+
+    /**
+     * Tests the request method
+     */
+    @Test
+    public void testRequest() {
+        CliDriver cli = new CliDriver();
```

Comment:
CliDrier and CliPresenter shouldnt be used in this test. I think it will be worthwhile making a presenter that can be used in tests. TestPresenter or something. TestPresenter can just keep a list of responses it is given and then allow test to access the responses for validation. Message me if this doesnt make sense-

Reply:
TestPresenter wouldnt need an InOut parameter in constructor, overall just simler for testing. Would save us time in long run.